--- a/Source/WebKit/Shared/glib/ProcessExecutablePathGLib.cpp	2026-10-08 16:36:27.803116296 +0800
+++ b/Source/WebKit/Shared/glib/ProcessExecutablePathGLib.cpp	2026-10-08 16:36:27.815116263 +0800
@@ -44,7 +44,6 @@
 
 static String findWebKitProcess(const ASCIILiteral processName)
 {
-#if ENABLE(DEVELOPER_MODE)
     static const char* execDirectory = g_getenv("WEBKIT_EXEC_PATH");
     if (execDirectory) {
         String processPath = FileSystem::pathByAppendingComponent(FileSystem::stringFromFileSystemRepresentation(execDirectory), processName);
@@ -52,6 +51,7 @@
             return processPath;
     }
 
+#if ENABLE(DEVELOPER_MODE)
     static String executablePath = getExecutablePath();
     if (!executablePath.isNull()) {
         String processPath = FileSystem::pathByAppendingComponent(executablePath, processName);
